fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryIndexPlanWithOverScanTest.java

```diff
@@ -248,8 +250,12 @@ private static RecordCursorContinuation assertSameResults(@Nonnull FDBRecordStor
             do {
                 indexResult = indexCursor.getNext();
                 overscanResult = overscanCursor.getNext();
-                assertEquals(indexResult.getContinuation().toByteString(), overscanResult.getContinuation().toByteString(), "Continuation byte strings should match");
-                assertArrayEquals(indexResult.getContinuation().toBytes(), overscanResult.getContinuation().toBytes(), "Continuation byte arrays should match");
```

Comment:
Does this assertion actually need to change? I would have thought that we'd expect the continuations to stay the same, even if both are wrapped in protobuf